linera-rpc/src/grpc_network.rs

```diff
@@ -463,6 +478,9 @@ where
             .await
         {
             Ok((info, actions)) => {
+                SERVER_REQUEST_LATENCY_PER_REQUEST_TYPE
+                    .with_label_values(&["handle_lite_certificate"])
+                    .observe(start.elapsed().as_secs_f64());
                 SERVER_REQUEST_SUCCESS
                     .with_label_values(&["handle_lite_certificate"])
                     .inc();
```

Comment:
This could now also be deduplicated.

```suggestion
                Self::log_request_success_and_latency(start, "handle_lite_certificate");
```

Reply:
True, will fix